core/datastorage/src/main/java/com/cognifide/aet/vs/mongodb/MongoDBClient.java

```diff
@@ -103,7 +102,17 @@ public void deactivate() {
 
     return companies;
   }
-  
+
+  protected void setupConfiguration(MongoDBClientConf config) {
+    if (StringUtils.isNoneBlank(config.mongoURI())) {
+      this.mongoUri = config.mongoURI();
+    } else {
+      this.mongoUri = StringUtils.defaultString(
```

Comment:
What about a case where the ENV varialbe is an empty string? If I'm not mistaken the `StringUtils.defaultString` only checks for null